docs/CONTRIBUTING.md

```diff
@@ -2,6 +2,12 @@
 
 If you find an issue with this package, please create an issue in GitHub. If you'd like, we welcome any contributions. Fork this repository and submit a pull request.
 
+## Adding new functionality or fixing relevant bugs
+
+If you are making any relevant changes that is worth communicating to our users, please include a note about it in our CHANGELOG.md. You can include it as part of the PR where you are submitting your changes.
```

Comment:
I might be more specific here. `CHANGELOG.md` should have the next minor version listed as `# v1.X.0 (Unreleased)` and any changes should go under there. If the next version is not listed, it would be encouraged for the contributor to add it. The one caveat being if it's a critical bug fix that needs to be released immediately, one would need to add the next patch version instead (so it's okay if you have two unreleased versions listed in the changelog). In addition we should provide an example of how our changelog is formatted:
```
# v1.X.X (Unreleased)

## Enhancements
* Brief Enhancement description by @contributer ([#XXX](https://github.com/hashicorp/go-tfe/pull/XXX))

## Bug Fixes
* Brief bug fix description by @contributer ([#XXX](https://github.com/hashicorp/go-tfe/pull/XXX))
```

On a side note, I'd love to create an action that automagically updates `CHANGELOG.md` to list the next release whenever a new release is created.